react/AppLinker/index.jsx

```diff
@@ -68,7 +68,7 @@ export class AppLinker extends React.Component {
 
       if (context) {
         return {
-          onClick: () => context.call('openApp', href),
+          onClick: () => context.call('openApp', href, app),
```

Comment:
nit: is it possible to add a test on this call, in order to make the code more robust?

Reply:
I tried, but with @acezard we don't know how to mock the `context` object (which is handled by the messaging service)

Reply:
ok; I tried without success as well.